server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -86,6 +93,10 @@ export class TenantManager {
                     return;
                 }
 
+                Lumberjack.info(`Invalid key. Deleting from cache.`, lumberProperties);
+                this.cache.delete(tenantId).catch((error) => {
+                    Lumberjack.error(`Unable to delete key from cache.`, { [BaseTelemetryProperties.tenantId]: tenantId }, error);
```

Comment:
Nit: Can we use lumberProperties instead of [BaseTelemetryProperties.tenantId]: tenantId?